### DISCUSSION TITLE
Uh yeah just wondering what this repo is for...

### DISCUSSION THREAD

**Discussion by TechStudent10** [microsoft/fluentui#17605] [Q&A] [answered]
So I just stumbled on this repo in the [Microsoft Topic](https://github.com/topics/microsoft) and I could just read the README (well it's literally begging me to read it) but I wanted to test the new GitHub Discussions tab while still getting some knowledge about this repo. Thanks in advance!

**Comment by khmakoto** [comment, marked as answer]
Hi @TechStudent11! Fluent UI is a library of UI components based on React that follows Microsoft's Fluent design language. It provides a number of components and utilities to ease building the UI of applications.

**Reply by TechStudent10** [reply]
@khmakoto thanks!

**Reply by khmakoto** [reply]
You're welcome!
